src/chipper/state.cljs

```diff
@@ -0,0 +1,148 @@
+(ns chipper.state
+  (:require [clojure.string :refer [split-lines]]
+            [chipper.notes :as notes]))  ; not good to require this in utils
+
+(defn reset-cursor! [state]
```

Comment:
hm, i should probably put all the state-changing stuff from keyboard into here too

Reply:
Or, really, there should be an `actions`